buildlib/dockers/ubuntu-release.Dockerfile

```diff
@@ -14,13 +16,15 @@ RUN apt-get update && \
         libcap2 \
         libnuma-dev \
         libtool \
+        libnvidia-compute-${DRIVER_VERSION} \
         make \
         maven \
         udev \
         wget \
         environment-modules \
         pkg-config \
-    && apt-get remove -y openjdk-11-* || apt-get autoremove -y \
+        sudo \
+    && apt-get remove -y openjdk-11-* cuda-compat* || apt-get autoremove -y \
```

Comment:
Why do you remove openjdk-11-* ?
Please add a comment why you remove cuda-compat*

Reply:
No worry about openjdk-11-* - as far as I see it existed before

Reply:
What is about 2nd part - Please add a comment why you remove cuda-compat*

Reply:
cuda-compat* is not a part of the default CUDA installation, thus we refrain from creating a dependency on it.